### PR TITLE
Update translate schemas task to look in the input folder first when resolving schemas.

### PR DESCRIPTION
**Pull request by karthikbalasub**
Improve logging when conflicts are detected during parsing. Message now specifies the location of the conflict instead of printing the schema. Having the location is more useful to resolve conflict errors.

### CODE REVIEW

**Comment by junchuanwang** [comment]
You might want to document somewhere why this order matters.

File: gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/tasks/TranslateSchemasTask.java [hunk 1]
```diff
@@ -44,7 +44,7 @@ public void translate()
     getProject().getLogger().info("Translating data schemas ...");
     _destinationDir.mkdirs();
 
-    String resolverPathStr = _resolverPath.plus(getProject().files(_inputDir)).getAsPath();
+    String resolverPathStr = getProject().files(_inputDir).plus(_resolverPath).getAsPath();
```
Comment (junchuanwang) [inline gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/tasks/TranslateSchemasTask.java:47]: You might want to document why this order matters.

Comment (karthikbalasub) [reply gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/tasks/TranslateSchemasTask.java:49]: Done

**Review by BrianPin — APPROVED** [review]

**Review by nickibi — APPROVED** [review]
LGTM

**Pull request merged**